lib/api/core/auth/passwordManager.js

```diff
@@ -0,0 +1,21 @@
+var crypto = require('crypto');
+
+module.exports = function PassportManager () {
+
+  this.secret = 'jbsB8r69PFP39KdLtjVr25Z22';
```

Comment:
Magic number. Has this specific value any significance? If so, can you add a comment to explain it?
If not, is it possible to generate it from a configuration parameter?